### PR TITLE
Initial setup

### LINKED ISSUE #8: Setup initial repository folders and files
- [x] Create pipeline folder
- [x] Create dashboard folder
- [x] Create `requirements.txt` within all the above folders
- [x] Create schema folder
- [x] Create terraform folder
- [x] Create diagrams folder
- [x] Create general 'requirements.txt`
- [x] Create `README.md`'s within folders

As engineers, we need to set up the initial folders and files for our Github repository to ensure we have thought through what folders are needed.

### PR DESCRIPTION
**Pull request by alina-101**
Created `pipeline`, `schema`, `terraform`, `dashboard` and `diagrams` folders as well as necessary `requirements.txt` files.
Added initial Architecture diagram image.

Closes #8

### CODE REVIEW

File: dashboard/requirements.txt
Comment (Zhi-704) [inline dashboard/requirements.txt:1]: Could we potentially add pylint, pytest?

**Comment by e-lemma** [comment]
Why is there a separate readme for the pipeline & dashboard?

File: diagrams/Apollo bandcamp.png
Comment (Zhi-704) [inline diagrams/Apollo bandcamp.png:1]: Would prefer a more consistent formatting such as having service names as capitals under each box

File: terraform/main.tf
Comment (e-lemma) [inline terraform/main.tf:1]: Perhaps add the other necessary terraform files, e.g. `output.tf`.

**Comment by alina-101** [comment]
> Why is there a separate readme for the pipeline & dashboard?
In the review of last week's project we had smaller `README.md`'s within our folders which the coaches liked and they said to keep it but provide more detail in the smaller ones on how to run that folder's scripts and then reference that within the larger `README.md`

**Comment by Zhi-704** [comment]
> > Why is there a separate readme for the pipeline & dashboard?
> 
> In the review of last week's project we had smaller `README.md`'s within our folders which the coaches liked and they said to keep it but provide more detail in the smaller ones on how to run that folder's scripts and then reference that within the larger `README.md`

No worries then, makes sense to me!

**Comment by e-lemma** [comment]
> > Why is there a separate readme for the pipeline & dashboard?
> 
> In the review of last week's project we had smaller `README.md`'s within our folders which the coaches liked and they said to keep it but provide more detail in the smaller ones on how to run that folder's scripts and then reference that within the larger `README.md`

Interesting, sounds like a good idea!

File: diagrams/Apollo bandcamp.png
Comment (alina-101) [reply diagrams/Apollo bandcamp.png:1]: I think it's more important to include what the service does as opposed to its name. Especially to stakeholders who may not be aware of the specific AWS name. I suggest we include the service names in the `README.md`

**Review by Zhi-704 — APPROVED** [review]

**Pull request merged**